src/GoTrueApi.ts

```diff
@@ -167,4 +180,53 @@ export default class GoTrueApi {
       return { data: null, error }
     }
   }
+
+  /**
+   * Handle server-side auth events and set/delete cookies.
+   * Works for Next.js & Express (requires cookie-parser middleware).
+   */
+  handleAuthEvent(req: any, res: any) {
```

Comment:
Thoughts on renaming this? It seems too generic and it isn't clear what `supabase.auth.handleAuthEvent()` will actually do.

Some ideas: `setServersideAuth()`, `setAuthCookies()`, ...? 

Or even more explicit - `setExpressAuth()`, `setNextjsAuth()` etc, which call the same function. This opens up the possibility of adding more frameworks - sort of like https://github.com/Swizec/useAuth

Reply:
I've renamed to `setAuthCookie` 👍